transcom/mymove
# [MB-16990] Update RequestedServiceItemsTable stories

## [MB-16990](https://dp3.atlassian.net/browse/MB-16990)

## Summary

We merged a bit out of order on these, but no worries, the actual application code changes for two stories were reviewed separately in https://github.com/transcom/mymove/pull/11356

Need for design review of unintended (but I think happy) consequences changing the ServiceItemUpdateModal are worth calling out. (See happo diff on this PR)

## Verification Steps for the Author

Originally this story was only intended to update the storybook examples, so reviewing that those meet the AC is the main concern

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### How to test

1. `yarn storybook`
2. go to http://localhost:6006/?path=/story/office-components-requestedserviceitemstable--default
3. Verify entry and departure date also show up 

## Screenshots

![image](https://github.com/transcom/mymove/assets/15805554/5f9307e2-cb06-49c1-9913-e0416977cbaf)


[MB-16990]: https://dp3.atlassian.net/browse/MB-16990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
